DATABASEconf.md

```diff
@@ -0,0 +1,44 @@
+
+Project uses Google's Firebase to store data and information.
+
+For setting up database on your local machine follow below mentioned steps
+
+- Go to [Firebase console](https://console.firebase.google.com/u/0/?utm_source=firebase.google.com&utm_medium=referral) start a new project
+- Go to build-> Authentication, Keep sign-in method as email and password
```

Comment:
After starting a new project there's need to create/register the web app and while registering we can get all the configurations we want. But we have to enable them as mentioned in line 7 and 8

Reply:
Soo what steps should I add in it?

Reply:
if we copy the credentails beofre starting the service, the database key is not added in the config, that's why I mentioned to access the configurations(API KEY) in later stage

Reply:
> Soo what steps should I add in it?

add creating/registering app step in between line 6 and line 7

Reply:
> if we copy the credentails beofre starting the service, the database key is not added in the config, that's why I mentioned to access the configurations(API KEY) in later stage

Yes, that's okay

Reply:
> if we copy the credentails beofre starting the service, the database key is not added in the config, that's why I mentioned to access the configurations(API KEY) in later stage

Yes, that's okay